parmed/gromacs/gromacsgro.py

```diff
@@ -221,7 +222,33 @@ def write(struct, dest, precision=3, nobox=False):
             is True, no box will be written. If False, the periodic box will be
             defined to enclose the solute with 0.5 nm clearance on all sides. If
             periodic box dimensions *are* defined, this variable has no effect.
+        match_topology : bool, optional
```

Comment:
What's the use case for `match_topology=False` (which is the default)?  Such a file does not seem useful to me.  I would suggest either:

1. Remove the `match_topology` argument altogether and just keep the logic for the `True` case
1. Change the default to `True`.

I'd prefer 1 because then it could simplify the code and ease maintenance.  But if there's a compelling reason, choice 2 is OK.

Reply:
The purpose of ``match_topology=False`` was mainly to preserve the default behaviour for backwards compatibility. I have no specific use cases in mind for it so happy to remove.

Reply:
Then I’d vote to remove the option and simplify.